iml-manager-cli/src/server.rs

```diff
@@ -23,6 +23,13 @@ use std::{
 use structopt::StructOpt;
 use tokio::time::delay_for;
 
+#[derive(StructOpt, Debug)]
+pub struct RemoveHosts {
+    /// The host(s) to update. Takes a hostlist expression
+    #[structopt(short = "h", long = "hosts")]
```

Comment:
Might conflict with `-h` for help.

Reply:
Changed to `-d` and `deleted_hosts`

Reply:
Maybe use positional arguments instead of options: `... remove <host expression>`?

Reply:
I only reproduce code of @jgrund, so I think it may be generally and better than now. I can create pr for change adding hosts too.